xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-test/xwiki-platform-image-lightbox-test-docker/src/test/it/org/xwiki/image/lightbox/test/ui/LightboxIT.java

```diff
@@ -100,6 +100,21 @@ void openImageWithoutDescription(TestUtils testUtils, TestReference testReferenc
         // Make sure that the images are displayed.
         lightboxPage.reloadPage();
 
+        // Verify the image ID popover actions.
+        Optional<ImagePopover> imagePopover = lightboxPage.hoverImage(0);
+        if (imagePopover.isPresent()) {
```

Comment:
Isn't the popover expected? If the purpose of the test is to check the popover actions then the test should ensure that the popover is visible. If it's not then the test should fail.